### ISSUE TITLE
Database of Modal Typology

### ISSUE THREAD

**Issue by LinguList** [cldf-datasets/cldf-datasets#49]
This dataset provides data in CLDF format: https://github.com/CLMBRs/modal-typology. We could actually fork it to `cldf-datasets`, to embrace it as a cldf dataset?

They seem to have some questions in their code, where one could also help (https://github.com/CLMBRs/modal-typology/blob/master/cldf-format/cldfbench_modaltypology.py).

**Comment by LinguList** [comment]
Their code could also be a bit refactored, as they do not use `pycldf` to load the CLDF datasets: https://github.com/CLMBRs/modal-typology/blob/85907042e789706a00041e1080722a6eeff22b6c/cldf-format/modalscommands/iff.py#L9-L13

**Comment by LinguList** [comment]
Database with link to paper is here: https://clmbr.shane.st/modal-typology/

**Comment by xrotwang** [comment]
I was already in contact with the creator.

**Comment by LinguList** [comment]
Ah, and will they make a version of it?

**Comment by xrotwang** [comment]
Ah release you mean? I'd guess so. We can always ask.

**Comment by LinguList** [comment]
Because we could then also include it on Zenodo in our collection of cldf datasets, right?

**Comment by LinguList** [comment]
Or we fork and release via cldf-datasets. I was thinking of things we discussed for lexibank, if somebody is doing a lexibank dataset themselves, to develop some procedure.

**Comment by xrotwang** [comment]
I think for datasets that are under development and haven't had publications citing them, we should leave the decision about completeness for a first release to the original curators.

**Comment by LinguList** [comment]
But this has a paper citing it now, that's why I added it here.

**Comment by LinguList** [comment]
https://sigtyp.github.io/workshops/2022/sigtyp/papers/SIGTYP2022_proceedings.pdf#page=54

**Comment by xrotwang** [comment]
Ok, I'll talk to Shane. I think that even if we forked this repos to archive a release, there should be at least a tag marking the state of the data for the paper in the original repository.

**Comment by LinguList** [comment]
Yes. the official publication is July 14th, but they posted the proceedings earlier.
